fixture-monkey-jakarta-validation/src/main/java/com/navercorp/fixturemonkey/jakarta/validation/introspector/JakartaValidationJavaArbitraryResolver.java

```diff
@@ -117,17 +116,31 @@ public Arbitrary<String> strings(
 		return arbitrary
 			.filter(it -> {
 				if (!notBlank) {
-					return true;
+					if (it == null) {
+						return true;
+					} else {
+						return !containsControlCharacters(it);
+					}
```

Comment:
I implemented it in the simplest way possible, but if there is a better way or any recommendation, feel free to suggest it 🙂